envergo/pages/views.py

```diff
@@ -18,6 +18,100 @@ class HomeView(MoulinetteMixin, FormView):
     template_name = "pages/home.html"
 
 
+class GeometriciansView(MoulinetteMixin, FormView):
+    template_name = "pages/geometricians.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["testimonials"] = [
```

Comment:
Je disais plus tôt… Ces données sont purement des données de présentation. Est-ce que tu avais une raison particulière de les stocker dans du code python que plutôt directement dans un template ?

Reply:
Toute les données que j'ai mis dans le contexte sont des listes dont chaque item doit etre afficher de la même facon.
Pour faire ça, je voyais 3 options:
1. dupliquer le code pour chaque item:
```
<container blah blah>
    item 1
</container>
<container blah blah>
    item 2
</container>
<container blah blah>
    item 3
</container>
```
2. faire un template dédié : 
```
{% include 'container.html' item1 %}
{% include 'container.html' item2 %}
{% include 'container.html' item3 %}
```
3. faire une boucle for
```
{% for item in items %}
    <container blah blah>
        item 
    </container>
{% endfor %}
```

l'option 1 est trés désagréable en cas de modification ou de maintenance
l'option 2 ajoute des composants trop spécifique a mon besoin pour que cela ait du sens
l'option 3 à donc eu ma préférence.
Peut etre peut-on créer des liste sur lesquelles iterer directment dans le template,si c'est le cas je ne sais pas le faire, j'ai donc utiliser le contexte pour cela.

Reply:
Personnellement, quand il s'agit d'éléments de présentation simples, avec logique d'affichage purement statique,  en petit nombre et probablement pas amené à être modifiés souvent, je trouve que la simplicité de lecture de l'option 1 l'emporte sur les autres considérations.

Par exemple, avec la solution actuelle, si je devais modifier un témoignage, j'irais sur le template, je devrais lire du code pour comprendre la logique, puis trouver l'emplacement dans le views.py, etc. Alors qu'avec l'option 1, j'ouvre le template, un search & replace et basta !

Ceci dit, on est ici dans le domaine du subjectif, donc je m'en remet à ton jugement.